src/query-builder/SelectQueryBuilder.ts

```diff
@@ -1605,8 +1607,9 @@ export class SelectQueryBuilder<Entity extends ObjectLiteral>
 
     /**
      * Gets all raw results returned by execution of generated query builder sql.
+     * @param {Date} [timestamp] A timestamp to get datasets at a specific time.
      */
-    async getRawMany<T = any>(): Promise<T[]> {
+    async getRawMany<T = any>(timestamp?: Date): Promise<T[]> {
```

Comment:
timestamp must be set via query builder methods (with a better name to understand purpose of this timestamp).

Reply:
Same comment applies to all other methods below.

Reply:
The parameter is now part of the `FindOneOptions` and is no longer necessary here.